content/english/chatbot/_index.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Create your own chatbot"
+description: "An introductory guide to chatbots"
+date: 2020-02-217T11:45:38-07:00
+prereq: "None"
+difficulty: "Beginner"
+download: ""
+draft: false
+icon: "fas fa-code"
+---
+## Introduction
+In this workshop, you will learn how to create your own chatbot. By the end of the workshop, you will have a chatbot that can have a simple conversation with the user. You will learn how to use AIML, a language for creating chatbots. You will also learn how to use Pandorabots, a platform for hosting chatbots.
+
+## What is a chatbot?
+A chatbot is a computer program that simulates human conversation to answer questions. 
+
+## Examples of chatbots
+Chances are you've already interacted with a chatbot as part of your daily life. 
+
+You might use a virtual assistant to set a reminder on your phone, or to play a song on your smart speaker.

```

Comment:
Add image alt text-

Two women talking to each other with speech bubbles in the background.